src/ui/common/src/components/integrations/dialogs/snowflakeDialog.tsx

```diff
@@ -24,6 +25,18 @@ export const SnowflakeDialog: React.FC<Props> = ({
   value,
   editMode,
 }) => {
+  const [schema, setSchema] = useState<string>(
+    value?.schema ?? Placeholders.schema
+  );
+
+  useEffect(() => {
+    if (schema) {
+      onUpdateField('schema', schema);
+    } else {
```

Comment:
Is it ever possible for the length of schema to be 0?

From glancing at this code, it looks like the answer is no, but just making sure.

Reply:
Nope there should always be a schema, so if its blank, we want to use the default `public` one.